### PR TITLE
Add FastAPI example for python docs

### PR DESCRIPTION
**Pull request by estebanx64**
## Description
Provide an example with one of the most used and modern python apps/frameworks today `FastAPI` to replace the old Flask example

Replaced the old flask app by a new modern FastAPI since FastAPI is more popular nowadays, modern and use the latest python types features. This change was requested and approved by Craig and Eva

## Related issues or tickets
N/A

## Reviews

N/A

@craig-osterhout 
@usha-mandya

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by falconcr — APPROVED** [review]
LGTM

File: content/language/python/deploy.md [hunk 1]
```diff
@@ -75,16 +164,31 @@ To learn more about Kubernetes objects, see the [Kubernetes documentation](https
...
 
-   ```shell
+   ```console
+   deployment.apps/postgres created
+   service/postgres created
+   persistentvolumeclaim/postgres-pvc created
+   secret/postgres-secret created
+   ```
+
+   and let's deploy our python application
```
Comment (craig-osterhout) [inline content/language/python/deploy.md:183]:
```suggestion
  Now, deploy your python application.
```

**Review by craig-osterhout — CHANGES_REQUESTED** [review]
Thanks @estebanx64. Added some minor nits, otherwise LGTM.

File: content/language/python/containerize.md [hunk 2]
```diff
@@ -115,10 +174,10 @@ USER appuser
 COPY . .
 
 # Expose the port that the application listens on.
-EXPOSE 8000
+EXPOSE 5000
 
 # Run the application.
-CMD python3 -m flask run --host=0.0.0.0 --port=8000
+CMD python3 -m uvicorn app:app --host=0.0.0.0 --port=5000
```
Comment (dvdksn) [inline content/language/python/containerize.md:180]: mac users would, by default, already have a service listening on port 5000. Could we maybe choose a different high port, e.g. 8000

File: content/language/python/containerize.md [hunk 3]
```diff
@@ -231,34 +317,37 @@ directory.
...
 in a terminal.
 
 ```console
 $ docker compose up --build -d
 ```
 
-Open a browser and view the application at [http://localhost:8000](http://localhost:8000).
+Open a browser and view the application at [http://localhost:5000](http://localhost:5000).
+
+To see the OpenAPI docs you can go at [http://localhost:5000/docs](http://localhost:5000/docs)
```
[the remark this answers is not in the document]
Comment (estebanx64) [reply content/language/python/containerize.md]: Done!

File: content/language/python/deploy.md (hunk 1 above)
Comment (estebanx64) [reply content/language/python/deploy.md]: Done!

File: content/language/python/containerize.md (hunk 2 above)
Comment (estebanx64) [reply content/language/python/containerize.md]: Good catch, I didn't know there is a service running in Mac OS on 5000. I fix it to 8000. Thanks for the hint

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**